src/services/infra/StepFunctionsService.ts

```diff
@@ -0,0 +1,22 @@
+import { StepFunctions } from "aws-sdk"
+
+import { SiteLaunchMessage } from "../../../microservices/site-launch/shared/types"
+
+export default class StepFunctionsService {
+  private client: StepFunctions
+
+  constructor(private stateMachineArn: string) {
+    this.client = new StepFunctions({ region: "ap-southeast-1" })
```

Comment:
should the region come from env vars?

Reply:
yes, modified here ca073d14d428b2416659ed0b217655809bbf1214

Reply:
@harishv7 out of curiosity - why inject it from env vars? my understanding is that it's a constant (as we aren't gonna change `region`) so i'd be more inclined to export it from a `constants` file

Reply:
@seaerchin I am fine with a constants file too. I suggested env vars, because we are building this open source, for people to try this, they would need to know exactly what are the **config** they need to provide. Which would be better to come from an env-var or an explicit `config.js` file